src/http/RegisteredHeadersHash.cci

```diff
@@ -41,7 +41,7 @@
  * Please see the COPYING and CONTRIBUTORS files for details.
  */
 #line 24 "RegisteredHeadersHash.gperf"
-struct HeaderTableRecord;
+class HeaderTableRecord;
```

Comment:
We should not be hand-editing generated files. AFAICT, this needs to be fixed in `src/http/RegisteredHeadersHash.gperf` (which also has a "generated from..." header, but _that_ header is lying IIRC).

You will need a matching gperf version to properly regenerate everything. See `git grep gperf scripts/source-maintenance.sh` for a regeneration command.

Reply:
Ha! I originally fixed it in the gperf file, noticed it said "generated from", then moved it.